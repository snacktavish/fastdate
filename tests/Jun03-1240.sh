~/bin/speed-dating-master/src/fastdate --method_nodeprior --grid 3000 --prior_file Jun03-1240-50nodes.txt --bd_mu 1 --bd_lambda 2 --max_age 90  --bd_rho  1 --tree_file Jun03-1240.tre --rate_mean 0.0168 --rate_variance 0.0018 --out_file Jun03-1240.out
