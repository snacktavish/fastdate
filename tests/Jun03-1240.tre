(((((t24:1.57666006e-05,(t45:0.001396957919,t46:0.08618291219):0.009874767542):0.001565798393,(t27:4.187197061e-07,t28:0.03376216681):3.715378439e-15):1.538720078e-07,((((((((((t95:1.162956662e-06,t96:1.132496621e-06):0.00659611394,t39:0.005695939904):0.07446038881,(((t84:0.01675029339,t85:0.0002292230325):0.03118654075,t29:0.0003601704379):0.0008783027368,((t56:0.002096664653,(t78:0.0001445702648,t79:0.0002114057122):0.001104652153):0.0001581303694,(t21:0.4178386222,t22:1.210554799e-08):0.003552517101):0.00177405518):0.006337976993):0.04998454498,(t48:2.132972591e-07,t49:0.002040121627):3.746959297e-05):0.03752544985,(((((t59:2.354780839e-05,t60:3.368076892e-06):6.769043729e-06,(t38:0.000239893242,(t43:8.475156457e-05,t44:5.091482088e-12):0.002576066405):0.01350647374):1.23280573e-07,((((t12:0.02602265199,t13:0.02180611772):1.320737637e-09,(t37:0.01252938637,(t40:0.001801038741,t41:0.02438697582):9.696816323e-06):0.03784496804):0.01466120514,t11:9.845801404e-05):1.600185316e-07,t6:0.2488923752):9.046871921e-06):0.00164330228,(t34:2.346344596e-06,t35:3.344402337e-05):0.009238048856):0.00297799802,(((t33:0.003677132511,(t80:8.874387208e-06,t81:3.6659841e-05):9.160844628e-06):6.862120119e-05,((t4:0.4203356724,t16:2.476546804e-06):2.9883132e-05,t3:0.02947708949):0.006124512329):3.174872278e-06,(t47:0.008136885842,(t63:0.1107678532,t64:0.008755593094):2.492148657e-05):2.501730647e-05):0.002872498566):0.000257831841):2.275726376e-06,(t2:4.313639514e-06,((((t88:1.064968089e-05,(t99:2.805919476e-05,t100:2.87971471e-17):7.349696993e-10):0.0008909110796,t55:1.46583075e-07):0.08167967707,((t30:9.925667043e-06,t31:0.09962325499):0.07883437034,((t93:3.635105429e-05,t94:2.335161442e-06):1.088590548e-07,t50:5.568193081e-08):0.130452488):0.01622694327):5.672122442e-05,(t26:0.01518845256,t23:0.03063300571):0.004232977212):0.0005444778993):0.005975181905):0.03558022746,(t76:9.055929354e-05,t77:0.004715126459):0.1984622775):0.000687002847,((((t89:0.003011897434,t90:0.0001265254685):1.142067954e-12,((t82:0.004118307811,t83:0.002419536526):1.73232961e-09,(t91:8.614598131e-09,t92:0.0005943009536):9.928424811e-05):0.01586800631):0.006119877441,t42:9.426547802e-14):0.00872905083,(((t86:3.196848296e-07,t87:2.474532375e-06):0.008960322386,t67:0.00969222227):0.06628032058,t7:1.252733265e-05):0.04690802788):4.389343136e-05):3.924073572e-05,t8:8.64066246e-05):6.904144743e-18,(((t14:1.851982456e-07,(((t61:0.009739950844,t62:0.1361209645):9.816607963e-06,t36:0.1153935481):0.02860528071,((t57:0.01112518042,t58:1.414821672e-06):0.1296667718,(t65:0.0590565393,t66:0.01421077912):0.0614174786):4.733113205e-08):0.05034840266):0.02502574433,t9:0.001222533829):0.0005331575368,t15:4.42101277e-08):1.779237762e-08):0.0007013419979):0.0007485214673,(((t73:8.776524267e-06,(t74:0.0004061751563,t75:0.0007929281224):2.511389817e-06):0.06146455989,t17:0.03802905075):4.108670032e-07,(((t18:4.107122569e-05,(((t52:0.003126328103,t53:0.004475103501):0.004040993145,(t70:1.02482447e-05,t71:0.001306608092):0.000192803398):0.004769806871,t19:0.001010432676):1.613673865e-11):0.03250614183,(t97:3.997423302e-05,t98:8.804403212e-05):0.0007578924321):0.02956524173,(((t5:9.685613546e-17,(t20:2.739833405e-05,(t68:9.1762676e-07,t69:0.0003575494707):0.03398296873):0.16398667):0.008340407298,((t25:0.02145092288,(t32:6.578671903e-09,t54:2.843613421e-09):1.937464955e-08):1.532007554e-06,(t51:0.1893934037,t72:1.337187456e-05):7.025263053e-09):0.001050612959):0.001006111988,t1:0.01346023395):0.01551002178):1.943764716e-05):1.601362026e-06):0.004020468144,t10:1.35215508);
