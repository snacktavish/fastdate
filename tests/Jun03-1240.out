(((((t24:1.620540,(t45:0.900300,t46:0.900300)[&map_age=0.900300]:0.720240)[&map_age=1.620540]:1.320440,(t27:1.050350,t28:1.050350)[&map_age=1.050350]:1.890630)[&map_age=2.940980]:24.068023,((((((((((t95:0.630210,t96:0.630210)[&map_age=0.630210]:1.890630,t39:2.520840)[&map_age=2.520840]:4.051350,(((t84:0.960320,t85:0.960320)[&map_age=0.960320]:2.160720,t29:3.121040)[&map_age=3.121040]:2.550850,((t56:1.530510,(t78:0.630210,t79:0.630210)[&map_age=0.630210]:0.900300)[&map_age=1.530510]:2.970990,(t21:3.901300,t22:3.901300)[&map_age=3.901300]:0.600200)[&map_age=4.501501]:1.170390)[&map_age=5.671891]:0.900300)[&map_age=6.572191]:1.230410,(t48:1.650550,t49:1.650550)[&map_age=1.650550]:6.152051)[&map_age=7.802601]:6.602201,(((((t59:0.960320,t60:0.960320)[&map_age=0.960320]:2.850950,(t38:2.460820,(t43:1.440480,t44:1.440480)[&map_age=1.440480]:1.020340)[&map_age=2.460820]:1.350450)[&map_age=3.811270]:5.461821,((((t12:1.380460,t13:1.380460)[&map_age=1.380460]:4.951651,(t37:2.610870,(t40:0.990330,t41:0.990330)[&map_age=0.990330]:1.620540)[&map_age=2.610870]:3.721240)[&map_age=6.332111]:1.320440,t11:7.652551)[&map_age=7.652551]:0.780260,t6:8.432811)[&map_age=8.432811]:0.840280)[&map_age=9.273091]:0.750250,(t34:2.670890,t35:2.670890)[&map_age=2.670890]:7.352451)[&map_age=10.023341]:2.970990,(((t33:1.380460,(t80:0.600200,t81:0.600200)[&map_age=0.600200]:0.780260)[&map_age=1.380460]:2.040680,((t4:1.590530,t16:1.590530)[&map_age=1.590530]:0.600200,t3:2.190730)[&map_age=2.190730]:1.230410)[&map_age=3.421140]:8.912971,(t47:2.190730,(t63:1.200400,t64:1.200400)[&map_age=1.200400]:0.990330)[&map_age=2.190730]:10.143381)[&map_age=12.334111]:0.660220)[&map_age=12.994331]:1.410470)[&map_age=14.404802]:1.050350,(t2:5.371791,((((t88:0.810270,(t99:0.420140,t100:0.420140)[&map_age=0.420140]:0.390130)[&map_age=0.810270]:0.660220,t55:1.470490)[&map_age=1.470490]:1.920640,((t30:1.080360,t31:1.080360)[&map_age=1.080360]:1.440480,((t93:0.540180,t94:0.540180)[&map_age=0.540180]:0.660220,t50:1.200400)[&map_age=1.200400]:1.320440)[&map_age=2.520840]:0.870290)[&map_age=3.391130]:1.080360,(t26:3.691230,t23:3.691230)[&map_age=3.691230]:0.780260)[&map_age=4.471490]:0.900300)[&map_age=5.371791]:10.083361)[&map_age=15.455152]:1.920640,(t76:1.080360,t77:1.080360)[&map_age=1.080360]:16.295432)[&map_age=17.375792]:0.450150,((((t89:0.870290,t90:0.870290)[&map_age=0.870290]:1.860620,((t82:0.660220,t83:0.660220)[&map_age=0.660220]:0.810270,(t91:0.630210,t92:0.630210)[&map_age=0.630210]:0.840280)[&map_age=1.470490]:1.260420)[&map_age=2.730910]:5.581861,t42:8.312771)[&map_age=8.312771]:1.590530,(((t86:0.600200,t87:0.600200)[&map_age=0.600200]:0.930310,t67:1.530510)[&map_age=1.530510]:6.392131,t7:7.922641)[&map_age=7.922641]:1.980660)[&map_age=9.903301]:7.922641)[&map_age=17.825942]:0.780260,t8:18.606202)[&map_age=18.606202]:7.562521,(((t14:13.624542,(((t61:1.440480,t62:1.440480)[&map_age=1.440480]:1.470490,t36:2.910970)[&map_age=2.910970]:6.482161,((t57:1.500500,t58:1.500500)[&map_age=1.500500]:6.482161,(t65:1.440480,t66:1.440480)[&map_age=1.440480]:6.542181)[&map_age=7.982661]:1.410470)[&map_age=9.393131]:4.231410)[&map_age=13.624542]:2.430810,t9:16.055352)[&map_age=16.055352]:2.040680,t15:18.096032)[&map_age=18.096032]:8.072691)[&map_age=26.168723]:0.840280)[&map_age=27.009003]:17.765922,(((t73:0.810270,(t74:0.420140,t75:0.420140)[&map_age=0.420140]:0.390130)[&map_age=0.810270]:1.620540,t17:2.430810)[&map_age=2.430810]:14.794932,(((t18:5.131711,(((t52:1.170390,t53:1.170390)[&map_age=1.170390]:1.020340,(t70:0.930310,t71:0.930310)[&map_age=0.930310]:1.260420)[&map_age=2.190730]:2.100700,t19:4.291430)[&map_age=4.291430]:0.840280)[&map_age=5.131711]:5.851951,(t97:1.170390,t98:1.170390)[&map_age=1.170390]:9.813271)[&map_age=10.983661]:1.080360,(((t5:4.981661,(t20:3.841280,(t68:0.930310,t69:0.930310)[&map_age=0.930310]:2.910970)[&map_age=3.841280]:1.140380)[&map_age=4.981661]:0.840280,((t25:1.620540,(t32:0.660220,t54:0.660220)[&map_age=0.660220]:0.960320)[&map_age=1.620540]:3.301100,(t51:1.890630,t72:1.890630)[&map_age=1.890630]:3.031010)[&map_age=4.921641]:0.900300)[&map_age=5.821941]:0.840280,t1:6.662221)[&map_age=6.662221]:5.401801)[&map_age=12.064021]:5.161721)[&map_age=17.225742]:27.549183)[&map_age=44.774925]:6.872291,t10:51.647216)[&map_age=51.647216]:0.0;
